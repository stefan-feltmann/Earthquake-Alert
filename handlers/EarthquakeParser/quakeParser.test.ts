describe('QuakeParser', () => {
  test('', async () => {
    expect(true).toBeTruthy()
  })
})
