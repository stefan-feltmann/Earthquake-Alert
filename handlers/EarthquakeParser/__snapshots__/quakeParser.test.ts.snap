// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuakeParser  1`] = `
Object {
  "bbox": Array [
    -155.65049743652,
    19.218999862671,
    -1.7699999809265,
    -116.7945,
    63.4275,
    30.209999084473,
  ],
  "features": Array [
    Object {
      "geometry": Object {
        "coordinates": Array [
          -155.376007080078,
          19.2189998626709,
          30.2099990844727,
        ],
        "type": "Point",
      },
      "id": "hv72410077",
      "properties": Object {
        "alert": null,
        "cdi": null,
        "code": "72410077",
        "detail": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/detail/hv72410077.geojson",
        "dmin": null,
        "felt": null,
        "gap": 158,
        "ids": ",hv72410077,",
        "mag": 1.97000003,
        "magType": "md",
        "mmi": null,
        "net": "hv",
        "nst": 43,
        "place": "10 km E of Pāhala, Hawaii",
        "rms": 0.119999997,
        "sig": 60,
        "sources": ",hv,",
        "status": "automatic",
        "time": 1617412581640,
        "title": "M 2.0 - 10 km E of Pāhala, Hawaii",
        "tsunami": 0,
        "type": "earthquake",
        "types": ",origin,phase-data,",
        "tz": null,
        "updated": 1617412777350,
        "url": "https://earthquake.usgs.gov/earthquakes/eventpage/hv72410077",
      },
      "type": "Feature",
    },
    Object {
      "geometry": Object {
        "coordinates": Array [
          -155.145492553711,
          19.3328342437744,
          0.699999988079071,
        ],
        "type": "Point",
      },
      "id": "hv72410072",
      "properties": Object {
        "alert": null,
        "cdi": null,
        "code": "72410072",
        "detail": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/detail/hv72410072.geojson",
        "dmin": null,
        "felt": null,
        "gap": 106,
        "ids": ",hv72410072,",
        "mag": 1.75,
        "magType": "md",
        "mmi": null,
        "net": "hv",
        "nst": 38,
        "place": "14 km S of Fern Forest, Hawaii",
        "rms": 0.280000001,
        "sig": 47,
        "sources": ",hv,",
        "status": "automatic",
        "time": 1617412091060,
        "title": "M 1.8 - 14 km S of Fern Forest, Hawaii",
        "tsunami": 0,
        "type": "earthquake",
        "types": ",origin,phase-data,",
        "tz": null,
        "updated": 1617412292260,
        "url": "https://earthquake.usgs.gov/earthquakes/eventpage/hv72410072",
      },
      "type": "Feature",
    },
    Object {
      "geometry": Object {
        "coordinates": Array [
          -147.6683,
          63.4275,
          17.8,
        ],
        "type": "Point",
      },
      "id": "ak02149tbxzr",
      "properties": Object {
        "alert": null,
        "cdi": null,
        "code": "02149tbxzr",
        "detail": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/detail/ak02149tbxzr.geojson",
        "dmin": null,
        "felt": null,
        "gap": null,
        "ids": ",ak02149tbxzr,",
        "mag": 1.1,
        "magType": "ml",
        "mmi": null,
        "net": "ak",
        "nst": null,
        "place": "64 km E of Cantwell, Alaska",
        "rms": 1.02,
        "sig": 19,
        "sources": ",ak,",
        "status": "automatic",
        "time": 1617411550630,
        "title": "M 1.1 - 64 km E of Cantwell, Alaska",
        "tsunami": 0,
        "type": "earthquake",
        "types": ",origin,",
        "tz": null,
        "updated": 1617411774174,
        "url": "https://earthquake.usgs.gov/earthquakes/eventpage/ak02149tbxzr",
      },
      "type": "Feature",
    },
    Object {
      "geometry": Object {
        "coordinates": Array [
          -117.9245,
          38.169,
          11.1,
        ],
        "type": "Point",
      },
      "id": "nn00804207",
      "properties": Object {
        "alert": null,
        "cdi": null,
        "code": "00804207",
        "detail": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/detail/nn00804207.geojson",
        "dmin": 0.015,
        "felt": null,
        "gap": 107.54,
        "ids": ",nn00804207,",
        "mag": 0.3,
        "magType": "ml",
        "mmi": null,
        "net": "nn",
        "nst": 8,
        "place": "29 km SSE of Mina, Nevada",
        "rms": 0.06,
        "sig": 1,
        "sources": ",nn,",
        "status": "automatic",
        "time": 1617411091050,
        "title": "M 0.3 - 29 km SSE of Mina, Nevada",
        "tsunami": 0,
        "type": "earthquake",
        "types": ",origin,phase-data,",
        "tz": null,
        "updated": 1617411734044,
        "url": "https://earthquake.usgs.gov/earthquakes/eventpage/nn00804207",
      },
      "type": "Feature",
    },
    Object {
      "geometry": Object {
        "coordinates": Array [
          -155.650497436523,
          19.4838333129883,
          -1.76999998092651,
        ],
        "type": "Point",
      },
      "id": "hv72410057",
      "properties": Object {
        "alert": null,
        "cdi": null,
        "code": "72410057",
        "detail": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/detail/hv72410057.geojson",
        "dmin": null,
        "felt": null,
        "gap": 69,
        "ids": ",hv72410057,",
        "mag": 1.73,
        "magType": "ml",
        "mmi": null,
        "net": "hv",
        "nst": 33,
        "place": "22 km E of Honaunau-Napoopoo, Hawaii",
        "rms": 0.280000001,
        "sig": 46,
        "sources": ",hv,",
        "status": "automatic",
        "time": 1617410753800,
        "title": "M 1.7 - 22 km E of Honaunau-Napoopoo, Hawaii",
        "tsunami": 0,
        "type": "earthquake",
        "types": ",origin,phase-data,",
        "tz": null,
        "updated": 1617411084720,
        "url": "https://earthquake.usgs.gov/earthquakes/eventpage/hv72410057",
      },
      "type": "Feature",
    },
    Object {
      "geometry": Object {
        "coordinates": Array [
          -116.7945,
          33.4948333,
          4.67,
        ],
        "type": "Point",
      },
      "id": "ci39836744",
      "properties": Object {
        "alert": null,
        "cdi": null,
        "code": "39836744",
        "detail": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/detail/ci39836744.geojson",
        "dmin": 0.01816,
        "felt": null,
        "gap": 31,
        "ids": ",ci39836744,",
        "mag": 1.06,
        "magType": "ml",
        "mmi": null,
        "net": "ci",
        "nst": 47,
        "place": "9km NE of Aguanga, CA",
        "rms": 0.2,
        "sig": 17,
        "sources": ",ci,",
        "status": "automatic",
        "time": 1617410478440,
        "title": "M 1.1 - 9km NE of Aguanga, CA",
        "tsunami": 0,
        "type": "earthquake",
        "types": ",nearby-cities,origin,phase-data,scitech-link,",
        "tz": null,
        "updated": 1617410699950,
        "url": "https://earthquake.usgs.gov/earthquakes/eventpage/ci39836744",
      },
      "type": "Feature",
    },
    Object {
      "geometry": Object {
        "coordinates": Array [
          -117.6945,
          35.9266667,
          4.37,
        ],
        "type": "Point",
      },
      "id": "ci39836736",
      "properties": Object {
        "alert": null,
        "cdi": null,
        "code": "39836736",
        "detail": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/detail/ci39836736.geojson",
        "dmin": 0.0416,
        "felt": null,
        "gap": 85,
        "ids": ",ci39836736,",
        "mag": 0.91,
        "magType": "ml",
        "mmi": null,
        "net": "ci",
        "nst": 11,
        "place": "19km E of Little Lake, CA",
        "rms": 0.27,
        "sig": 13,
        "sources": ",ci,",
        "status": "automatic",
        "time": 1617410363900,
        "title": "M 0.9 - 19km E of Little Lake, CA",
        "tsunami": 0,
        "type": "earthquake",
        "types": ",nearby-cities,origin,phase-data,scitech-link,",
        "tz": null,
        "updated": 1617410577607,
        "url": "https://earthquake.usgs.gov/earthquakes/eventpage/ci39836736",
      },
      "type": "Feature",
    },
  ],
  "metadata": Object {
    "api": "1.10.3",
    "count": 7,
    "generated": 1617413526000,
    "status": 200,
    "title": "USGS All Earthquakes, Past Hour",
    "url": "https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_hour.geojson",
  },
  "type": "FeatureCollection",
}
`;
